src/app/forms/custom-sections/letters/letters.component.ts

```diff
@@ -73,11 +76,21 @@ export class LettersComponent implements OnInit, OnDestroy, OnChanges {
       }
       : undefined;
   }
+  // checking if the Technical Record History has current status code.
+  get checkForCurrentRecordInHistory(): boolean {
+    let ifCurrent = false;
```

Comment:
hasCurrent is a bit more appropriate for this variable name

Reply:
I will changed it now.